src/main/java/uk/gov/hmcts/sptribs/document/DocumentUtil.java

```diff
@@ -92,17 +94,17 @@ public static void uploadDocument(CaseData data) {
     public static List<String> validateUploadedDocuments(List<ListValue<CaseworkerCICDocument>> uploadedDocuments) {
         List<String> errors = new ArrayList<>();
 
-        if (uploadedDocuments != null && !uploadedDocuments.isEmpty()) {
+        if (!CollectionUtils.isEmpty(uploadedDocuments)) {
```

Comment:
CollectionUtils empty check can be used in all other methods in this file as well.